jtlibrary/python/jtmodules/src/jtmodules/separate_clumps.handles.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-version: 0.2.2
+version: 0.2.3
```

Comment:
Since interface has changed and pipeline would need to be modified, I would prefer a sharper version change.